mgr/src/server/plugins/volume_utils.cr

```diff
@@ -111,6 +112,21 @@ def validate_volume_create(req)
   NodeResponse.new(true, "")
 end
 
+def restart_shd_service(data)
+  services, _, _ = VolumeRequestToNode.from_json(data)
+
+  unless !services[GlobalConfig.local_node.id]?
```

Comment:
I think above `unless` and `  if !services[GlobalConfig.local_node.id]?.nil?` are same logic? Will replace with `if` for linter to pass.